src/main/java/reposense/util/FileUtil.java

```diff
@@ -122,8 +127,67 @@ public static void copyDirectoryFiles(Path src, Path dest) throws IOException {
         }
     }
 
+    /**
+     * Generates a list of paths of {@code fileType} contained in the given {@code directoryName} directory.
+     */
+    public static List<Path> listFilePath(String directoryName, String fileType) {
```

Comment:
`listFilePath` kinda sounds like we are printing out the file paths, which is not what we are doing here. Perhaps rename this as `getFilePaths`? 
Also, since this seems to just a helper method within `FileUtil`, can make it private.